src/components/questions/QuestionList.jsx

```diff
@@ -4,7 +4,7 @@ import styles from './questions.module.css';
 
 const QuestionList = (props) => {
 
-  let firstTwoQuestions = props.data.results.slice(0, 2);
+  let firstTwoQuestions = props.data.results.slice(0, 4);
```

Comment:
If this is a permanent change you might want to consider updating the variable name to reflect it is the first four questions not the firs two.